virt/logging_events_monitoring/virt-collecting-virt-data.adoc

```diff
@@ -1,25 +1,54 @@
 
 toc::[]
 
-When opening a support case, it is helpful to provide debugging
-information about your cluster to Red Hat Support.
+When you submit a xref:../../support/getting-support.adoc#support-submitting-a-case_getting-support[support case] to Red Hat Support, it is helpful to provide debugging information for {product-title} and {VirtProductName} by using the following tools:
 
-The `must-gather` tool enables you to collect diagnostic information about your
-{product-title} cluster, including virtual machines and other data related to
-{VirtProductName}.
+must-gather tool::
+The `must-gather` tool collects diagnostic information, including resource definitions and service logs.
 
-For prompt support, supply diagnostic information for both {product-title}
-and {VirtProductName}.
+Prometheus::
+Prometheus is a time-series database and a rule evaluation engine for metrics. Prometheus sends alerts to Alertmanager for processing.
 
-include::modules/about-must-gather.adoc[leveloffset=+1]
+Alertmanager::
+The Alertmanager service handles alerts received from Prometheus. The Alertmanager is also responsible for sending the alerts to external notification systems.
 
-include::modules/virt-about-collecting-virt-data.adoc[leveloffset=+1]
+include::modules/virt-collecting-data-about-your-environment.adoc[leveloffset=+1]
 
-include::modules/gathering-data-specific-features.adoc[leveloffset=+1]
+[id="additional-resources_collecting-data-about-your-environment"]
+[role="_additional-resources"]
+=== Additional resources
 
-include::modules/virt-must-gather-usage-targeted-vm-data.adoc[leveloffset=+1]
+* Configuring the xref:../../monitoring/configuring-the-monitoring-stack.adoc#modifying-retention-time-for-prometheus-metrics-data_configuring-the-monitoring-stack[retention time] for Prometheus metrics data.
+* Configuring the Alertmanager to send xref:../../monitoring/managing-alerts.adoc#sending-notifications-to-external-systems_managing-alerts[alert notifications] to external systems.
+* Collecting `must-gather` data for xref:../../support/gathering-cluster-data.adoc#support_gathering_data_gathering-cluster-data[{product-title}].
+* Collecting `must-gather` data for link:https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.10/html-single/troubleshooting_openshift_data_foundation/index#downloading-log-files-and-diagnostic-information_rhodf[{rh-storage-first}].
+* Collecting `must-gather` data for xref:../../virt/logging_events_monitoring/virt-collecting-virt-data.adoc#gathering-data-specific-features_virt-collecting-virt-data[{VirtProductName}].
+* Collecting Prometheus metrics for xref:../../monitoring/managing-metrics.adoc#querying-metrics-for-all-projects-as-an-administrator_managing-metrics[all projects] as a cluster administrator.
+
+include::modules/virt-collecting-data-about-vms.adoc[leveloffset=+1]
+
+[id="additional-resources_collecting-data-about-vms"]
+[role="_additional-resources"]
+=== Additional resources
+
```

Comment:
Nit: again, not totally sure if this is required/new for Jupiter, but the contrib guide shows *no* newline between the add'l resources subheading and the unordered list: https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#additional-resources-sections

Reply:
Now I remember seeing that and wondering whether it was an error. IIRC, not leaving a line between a header and following text sometimes caused rendering problems. However, evidently that does not happen with unordered lists, so I will follow that practice. In fact, I think I will create a snippet so that my aged brain cells don't overheat with all these details. :-)